proposals/p2240.md

```diff
@@ -0,0 +1,670 @@
+# Variadics
```

Comment:
How so?

Reply:
_pack_ and following things in the proposal seems more general than variadics, and it occupies quite a lot.

Reply:
Huh. Packs aren't intended to be more general than variadics -- are you saying there are plausible non-variadic uses of them? If you're saying they aren't _necessary_ for variadics, then you might well be right (see the note on line 223), but I'm not sure yet, so I'm leaving them in the draft for now because I think I refer to them elsewhere in the doc.

Reply:
Are we talking about the variadic template/macro as the comparison or some math abstraction of variadic?